test/chainrules.jl

```diff
@@ -62,12 +56,8 @@
                 test_scalar(loggamma, x)
 
                 isreal(x) || continue
-
-                Δx, x̄ = randn(2)
-                Δz = (randn(), randn())
-
-                frule_test(logabsgamma, (x, Δx))
-                rrule_test(logabsgamma, Δz, (x, x̄))
+                test_frule(logabsgamma, x)
+                test_rrule(logabsgamma, x; output_tangent=(randn(), randn()))
```

Comment:
Not changed anywhere upstram, but an old discrepancy i notice now:
technically this should be
```suggestion
                test_rrule(logabsgamma, x; output_tangent=Composite{Tuple{Float64, Float64}}(randn(), randn()))
```
but that is kinda long.
and it passes without it cos apparentlyt the rrule only cares that its input is iterable.